backend/coreapp/flags.py

```diff
@@ -125,6 +129,10 @@ def to_json(self) -> Dict[str, Union[str, list[str]]]:
     Checkbox(id="mwcc_debug_on", flag="-g"),
 ]
 
+COMMON_PPC_OBJDUMP_FLAGS: Flags = [
+    # FlagSet(id="ppc_microprocessor", flags=["", "-M gekko", "-M broadway"]),
```

Comment:
Why is this code commented out?

Reply:
Simon said the gc/wii configuration tab already had too much stuff and options, so I ended up commenting this out.
Not sure what we will end up deciding about this

Reply:
removed, since it isn't necessary for this to be configurable - I added it as a hard flag for the wii/gc preset